riverdriver/riversql/go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/riverqueue/river/riverdriver/riversql
```

Comment:
I have some qualms with this package name. Primarily worried that it could be confusing or take away a name that we might want to use for other purposes. However the best alternative I can come up with is `riverdbsql` which I don't love either 😕

Reply:
@bgentry 

I'm -1 on `riverdbsql` just because it doesn't really reflect the actual naming of `database/sql` very well (i.e. abbreviation in one but not the other).

I just pushed a change that renames it to `riverdatabasesql` — it's a little bit of a mouthful (one reason I originally avoided it), but I figure that this package will be used so rarely that it probably doesn't matter. What do you think?